### PR TITLE
[stable/ethereum] Upgrade geth and switch to PoA mining

### PR DESCRIPTION
**Pull request by kigawas**
#### What this PR does / why we need it:
Upgrade geth to v1.8 and switch to PoA mining for the sake of saving CPU resource in private networks.

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [X] Chart Version bumped
- [X] Variables are documented in the README.md
- [X] Title of the PR starts with chart name (e.g. `[stable/chart]`)

### CODE REVIEW

File: stable/ethereum/templates/bootnode.deployment.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-apiVersion: apps/v1beta2 
+apiVersion: apps/v1beta2
```
Comment (davidkarlsen) [inline stable/ethereum/templates/bootnode.deployment.yaml]: can be apps/v1

Comment (kigawas) [reply stable/ethereum/templates/bootnode.deployment.yaml]: changed

File: stable/ethereum/templates/geth-miner.deployment.yaml [hunk 2]
```diff
@@ -27,9 +27,9 @@ spec:
         image: {{ .Values.geth.image.repository }}:{{ .Values.geth.image.tag }}
         imagePullPolicy: {{ .Values.imagePullPolicy }}
         command: ["/bin/sh"]
-        args: 
+        args:
         - "-c"
-        - "geth --bootnodes=`cat /root/.ethereum/bootnodes` --mine --etherbase=0 --networkid=${NETWORK_ID} --ethstats=${HOSTNAME}:${ETHSTATS_SECRET}@${ETHSTATS_SVC} --verbosity=5"
```
Comment (kigawas) [inline stable/ethereum/templates/geth-miner.deployment.yaml:32]: https://github.com/ethereum/go-ethereum/blob/master/cmd/utils/flags.go#L1000

setting etherbase=0 is deprecated

File: stable/ethereum/values.yaml [hunk 3]
```diff
@@ -34,12 +34,13 @@ geth:
       rpcapi: 'eth,net,web3'
   miner:
     # miner nodes
-    replicaCount: 3
+    replicaCount: 1
```
Comment (kigawas) [inline stable/ethereum/values.yaml:37]: multiple miners on same address will race

**Pull request closed without merging**